### PR TITLE
Combine hybrid results

### PR DESCRIPTION
**Pull request by umbernhard**
This is a (for now) ugly script that runs the SUITE p-value computation on two independent audits (one ballot comparison and one ballot polling) to support California's two-step auditing procedure. This is a first step towards more broadly supporting stratified audits. 

I have tested it on a few hand-crafted test cases, but one thing I need to do before I merge is test it on some actual SUITE data to make sure it behaves the way I expect it to.

### CODE REVIEW

File: scripts/combine-hybrid-results.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    return suite.BallotComparisonStratum(
+        contest.total_ballots_cast, vote_counts, misstatements, sample_size,
+    )
+
+def ballot_polling_stratum(audit: Election, remap):
+    contest = audit.contests[0]
+    vote_counts = {choice.id: choice.num_votes for choice in contest.choices}
+
+    remapped_vote_counts = {remap[id1]: vote_counts[id1] for id1 in vote_counts}
+    suite_contest = sampler_contest.from_db_contest(contest)
```
Comment (jonahkagan) [inline scripts/combine-hybrid-results.py:29]: I think this is unused and can be removed

File: scripts/combine-hybrid-results.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    )
+    return suite.BallotComparisonStratum(
+        contest.total_ballots_cast, vote_counts, misstatements, sample_size,
+    )
+
+def ballot_polling_stratum(audit: Election, remap):
+    contest = audit.contests[0]
+    vote_counts = {choice.id: choice.num_votes for choice in contest.choices}
+
+    remapped_vote_counts = {remap[id1]: vote_counts[id1] for id1 in vote_counts}
```
Comment (jonahkagan) [inline scripts/combine-hybrid-results.py:28]: could probably just do the remapping the first time you define `vote_counts`

File: scripts/combine-hybrid-results.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    suite_contest = sampler_contest.from_db_contest(contest)
+
+    sample_results = contest_results_by_round(contest)
+    remapped_sample_results = {}
+
+    for rd in sample_results:
+        remapped_sample_results[rd] = {}
+        for id1 in sample_results[rd]:
+            remapped_sample_results[rd][remap[id1]] = sample_results[rd][id1]
```
Comment (jonahkagan) [inline scripts/combine-hybrid-results.py:37]: If you want to get fancy with dictionary comprehensions...
```suggestion
    sample_results = { 
      round_id: {
        remap[choice_id]: result
        for choice_id, result in round_results.items()
      }
      for round_id, round_results in contest_results_by_round(contest).items()
    }
```

**Review by jonahkagan — APPROVED** [review]
Looks great!

File: scripts/combine-hybrid-results.py (hunk 3 above)
Comment (umbernhard) [reply scripts/combine-hybrid-results.py:37]: ah I knew there was a way to do this I just couldn't figure it out.

**Pull request merged**